tests/melt_visco_plastic.cc

```diff
@@ -227,11 +189,11 @@ namespace aspect
          * MaterialModelOutputs object that is handed over, if it exists.
          * Does nothing otherwise.
          */
-        void fill_plastic_outputs (const unsigned int point_index,
-                                   const std::vector<double> &volume_fractions,
-                                   const bool plastic_yielding,
-                                   const MaterialModel::MaterialModelInputs<dim> &in,
-                                   MaterialModel::MaterialModelOutputs<dim> &out) const;
+        // void fill_plastic_outputs (const unsigned int point_index,
```

Comment:
Is there a compiler issue here with having both melt outputs and the plastic outputs? I recall running into something similar before.